examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Inc/provisioning.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __PROVISIONING_H__
+#define __PROVISIONING_H__
+
+#include "user_data.h"
+
+int8_t provisioning_start( user_data_t* dst );
```

Comment:
`provisioning_start` makes me think to call `provisioning_finish` at the end when I finish the provisioning process. But actually this isn't true, it is not needed. I'd call this function `do_provisioning` or something better which reflects more its behaviour.